core/src/switcher/switcher.tsx

```diff
@@ -28,6 +28,7 @@ export const Switcher = <T,>(props: Props<T>): JSX.Element => (
 					key={option.label || option.key}
 					icon={option.icon}
 					children={option.label}
+					highlight={selected}
```

Comment:
Wait.. this seems to be a little bit breaking change. I'm waiting for the deployment to see, but it seems like now the selected button would also be highlighted (blue)? If so, this should be optional.

Anyway, to confirm first, which is the one you want to achieve:

<img width="130" alt="image" src="https://user-images.githubusercontent.com/5953369/114338746-ee876500-9b7d-11eb-9a8a-5ccd689ecc78.png">

Reply:
I want to achieve it. the selected button is highlighted (blue)
<img width="200" alt="Screen Shot 2021-04-13 at 8 33 43 PM" src="https://user-images.githubusercontent.com/22491666/114561192-8b89f100-9c97-11eb-919a-8f542bb51947.png">

Reply:
and we can change the value of the switcher by clicking the button and highlight also change

Reply:
This looks like a toggle :))

Anyway, that is ok, but we should not change the current visual (which is only greyed out the selected button, not blued it) as it would harm the other users.

What should be done is:
- The Switcher component should have a "highlight" prop
- If that prop is true, the selected option should have "highlight=true"
- If that prop is false, the selected option should have "selected=true"